app/models/manageiq/providers/container_manager.rb

```diff
@@ -23,6 +24,12 @@ class ContainerManager < BaseManager
 
     virtual_column :port_show, :type => :string
 
+    supports :common_logging do
```

Comment:
> Not sure I like the phrase "common logging" as that is a Red Hat idiom. cc @blomquisg 

@Fryguy I can change all occurrences of `common_logging` to `kibana_support` . @simon3z is this OK with you?